src/main/java/org/jenkinsci/plugins/casc/integrations/GlobalMatrixAuthorizationStrategyConfigurator.java

```diff
@@ -0,0 +1,59 @@
+package org.jenkinsci.plugins.casc.integrations;
```

Comment:
I would rather create a package for the plugin. Role Strategy code could be moved to a package as well

Reply:
I added it to the demos folder instead.